### DISCUSSION TITLE
付款了却没有购买成功

### DISCUSSION THREAD

**Discussion by henmang** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#626] [General]
Hi, if you have a question about AlDente, please use the Discussions section of Github: https://github.com/davidwernhart/AlDente/

When purchasing the lifetime version, the purchase information is not returned, the email is not received, and the contact information cannot be found. What should I do?

**Comment by MatthiasKerbl** [comment]
Hi @henmang,

There are multiple potential reasons why you did not get your license code after the purchase:

- Sometimes it takes up to 30 Minutes for the purchase emails to go through.
- The purchase emails end up in the junk email folder.
- You had a typo in the email address during your purchase.
- Failures of the payment networks/providers.
- 
Please contact the support team of our payment provider Paddle at assist@paddle.com and include the email address you used to purchase AlDente Pro. The Paddle support will try to solve your issue as fast as possible.

Cheers,
Matthias

**Reply by henmang** [reply]
My email was not handled by everyone, only the robot replied to one. I provided information. No one is reading it.

**Comment by henmang** [comment]
24 hours have passed, no progress has been made, and no one has contacted me.
